examples/pl_basics/README.md

```diff
@@ -2,77 +2,7 @@
 
 Use these examples to test how Lightning works.
 
-## MNIST Examples
-
-Here are 5 MNIST examples showing you how to gradually convert from pure PyTorch to PyTorch Lightning.
```

Comment:
As noted by @Keiku, this MNSIT section is a complete duplicate of `examples/convert_from_pt_to_pl/README.md`, so I simply removed it from here.

Reply:
Thank you. For the user, I think it's a simple README.md.